javascript/week1/main.js

```diff
@@ -0,0 +1,98 @@
+// Age-ify (A future age calculator)
+let yearOfBirth = 1988; // it can change so "let" variable
+const yearFuture = 2027; // as the future year given so "const" variable
+let age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old on " + yearFuture + ".");
+
+//Goodboy-Oldboy (A dog age calculator)
+let dogYearOfBirth = 2017; // in human years and as it can change so "let" variable
+const dogYearFuture = 2027; // as the future year given so "const" variable
+let dogYear;
+let shouldShowResultInDogYears = false; //boolean type variable.
```

Comment:
Likewise `dogYearOfBirth` and `shouldShowResultInDogYears` can both be `const` here. `dogYear` is the only value which changes in the code, so has to be `let`.

Reply:
same thing here. Thanks for your feedback 👍